neuro_san/service/registries_watcher/periodic_updater/manifest_periodic_updater.py

```diff
@@ -36,7 +38,11 @@ def __init__(self, manifest_path: str, update_period_seconds: int):
         self.update_period_seconds: int = update_period_seconds
         self.logger = logging.getLogger(self.__class__.__name__)
         self.updater = threading.Thread(target=self._run, daemon=True)
-        self.observer: RegistryObserver = RegistryObserver(self.manifest_path)
+        if self.use_polling:
+            poll_interval: int = self.compute_polling_interval(update_period_seconds)
+            self.observer = RegistryPollingObserver(self.manifest_path, poll_interval)
+        else:
+            self.observer = RegistryEventObserver(self.manifest_path)
         self.tool_factory: ServiceToolFactoryProvider = \
```

Comment:
Select with observer we want to use.
For now, hard-coded to polling variant.

Reply:
The idea is that we would poll for Docker volumes, but we would use event observer for local file systems?